plugins/data-management/src/HierarchicalTrackSelectorWidget/components/tree/HamburgerMenu.tsx

```diff
@@ -93,22 +82,34 @@ export default observer(function HamburgerMenu({
 
   return (
     <>
-      <IconButton
-        className={classes.menuIcon}
-        onClick={event => setMenuEl(event.currentTarget)}
-      >
-        <MenuIcon />
-      </IconButton>
-
-      <JBrowseMenu
-        anchorEl={menuEl}
-        open={Boolean(menuEl)}
-        onMenuItemClick={(_, callback) => {
-          callback()
-          setMenuEl(undefined)
-        }}
-        onClose={() => setMenuEl(undefined)}
+      <CascadingMenuButton
         menuItems={[
```

Comment:
Add track is no longer at the top here -- subjective, but I think some users might get lost when looking for this option. Maybe we should consider restricting this hamburger menu to just be manipulating the listed options i.e. expand/collapse/sort, and have separate button / ui for manipulating tracks or connections.

One option would be to make the Add track FAB have the text "Add track" with the plus Icon and then allow the user to select adding a track or connection...could add a connections manager, etc.

Reply:
the discoverability of "Add track" is a persistent problem. I'd be happy to sort it back to the top but ya, the current 'state' of the system is that add track options are in multiple places (Fab, Hamburger menu, top level menu) in the UI because of this discoverability problem

Reply:
I was messing around with connections as well while exploring this and the UI for it is suuuuper clunky with the multi modals.